c++/src/kj/common.h

```diff
@@ -347,6 +347,14 @@ KJ_NORETURN(void unreachable());
 #define KJ_FALLTHROUGH
 #endif
 
+#if KJ_HAS_CPP_ATTRIBUTE(nodiscard)
+#define KJ_MUST_USE_RESULT [[nodiscard]]
```

Comment:
Is this different from `KJ_WARN_UNUSED_RESULT`?

Reply:
No, it's just the C++17 way of doing things.

Reply:
Looks like you don't need it anyway.